linkml/generators/graphqlgen.py

```diff
@@ -50,13 +59,34 @@ def visit_class_slot(self, cls: ClassDefinition, aliased_slot_name: str, slot: S
             slotrange = slotrange + "!"
         return f"\n    {lcamelcase(aliased_slot_name)}: {slotrange}"
 
+    def visit_enum(self, enum: EnumDefinition):
+        if enum.permissible_values:
+            permissible_values = []
+            for value in enum.permissible_values:
+                permissible_values.append(self.name_compatiblity.compatible(value))
+            values = "\n    ".join(permissible_values)
+        else:
+            raise NotImplementedError(
+                "Enumerations using `reachable_from` instead of `permissible_values` to "
+                + "specify permissible values is not supported yet!!"
+            )
+        out = f"enum {camelcase(enum.name).replace(' ','')}\n  {{\n    {values}\n  }}\n\n"
+        return out
+
 
 @shared_arguments(GraphqlGenerator)
 @click.command(name="graphql")
+@click.option(
+    "--do-not-fix-incompatibilities",
+    is_flag=True,
+    show_default=True,
+    help="If activated, no fixable issues will be fixed.",
```

Comment:
i probably would call this "--strict" with the description "Treat warnings about invalid names or schema elements as errors", or clarify that this is about naming only, as is I wouldn't know what it's referring to <3

Reply:
I like it, will apply it.